src/OneSpanSign.ts

```diff
@@ -0,0 +1,39 @@
+import FormData from 'form-data';
+
+import { Api } from './api';
+import {
+  CreatePackageRequestPayload,
+  CreatePackageResponsePayload,
+  DocumentMetadata,
+  UploadDocumentRequestPayload,
+} from './types';
+
+export class OneSpanSign {
+  constructor(private apiKey: string, private apiUrl: string) {}
+
+  public async createPackage(payload: CreatePackageRequestPayload): Promise<CreatePackageResponsePayload> {
```

Comment:
wouldn't it make sense for this method to simply return a `Package`?

Reply:
Unfortunately, this endpoint only returns an ID ([source](https://community.onespan.com/products/onespan-sign/sandbox#/Packages/api.packages.post))

Reply:
ah poop